extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -26,9 +26,9 @@ class ArticleComment {
 	/** @var array */
 	public $mMetadata;
 
-	private $mText;
-	private $mRawtext;
-	public $mHeadItems;
+	private $mText = false; // parsed HTML
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)